packages/accordions/src/elements/accordion/components/Header.tsx

```diff
@@ -38,6 +39,18 @@ const HeaderComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement
     type: 'button',
     index: sectionIndex
   });
+  const onHeaderFocus = (e: FocusEvent) => {
+    e.persist();
+
+    setTimeout(() => {
+      const isAccordionButton = e.target.getAttribute('data-garden-id') === buttonGardenId;
+      const isFocusVisible = e.target.getAttribute('data-garden-focus-visible');
+
+      if (isAccordionButton && isFocusVisible) {
+        setIsFocused(true);
+      }
+    }, 0);
+  };
```

Comment:
I think this may be a warp in git history 👽 because this was the original code within the `Accordion.Header` component being restored after it was removed early in this PR. Currently we don't modify this file at all, however this is something we can talk more about. Come to think of it, there is history I am unaware of of why we implemented in this fashion.